packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -194,15 +196,17 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
     this.primary = true,
     this.centerTitle,
     this.titleSpacing = NavigationToolbar.kMiddleSpacing,
+    this.titleHeight = kToolbarHeight,
```

Comment:
Would prefer that the default value for this parameter was null. That leaves open the possibility of adding toolbarHeight to AppBarTheme.

Reply:
In #48557, I did the same thing. But as suggested by @dnfield , I did it this way. This also looks clean and easy to understand.

Reply:
@HansMuller is more familiar with the Material system and theming - I would go with what he says here.

Reply:
Sorry for causing any confusion or extra work!